webpack.common.js

```diff
@@ -0,0 +1,81 @@
+const path = require("path");
+const MiniCssExtractPlugin = require('mini-css-extract-plugin')
+const CopyPlugin = require("copy-webpack-plugin")
+const BrowserSyncPlugin = require('browser-sync-webpack-plugin')
+const PACKAGE = require('./package.json')
+const theme = PACKAGE.theme
+
+module.exports = {
+  entry: {
+    screen: [`./src/Frontend/Themes/${theme}/src/Js/Index.js`, `./src/Frontend/Themes/${theme}/src/Layout/Sass/screen.scss`]
```

Comment:
Niet logischer om op aparte lijnen te zetten als er nog zouden bij komen?

Reply:
Scss kan geen aparte entry zijn. Andere optie is om de scss te importeren in index.js, maar het leek me duidelijker om dat in de webpack.config te houden. Je kan een tweede lijn bijzetten met een js file en eventueel scss file, die dan ook een aparte js file als output heeft.

Reply:
Ik bedoelde vooral:

```
dule.exports = {
  entry: {
    screen: [
      `./src/Frontend/Themes/${theme}/src/Js/Index.js`,
      `./src/Frontend/Themes/${theme}/src/Layout/Sass/screen.scss`
   ]
````